code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -276,8 +276,7 @@
 	return ..()
 
 /obj/effect/spawner/lootdrop/maintenance/proc/hide()
-	invisibility = INVISIBILITY_OBSERVER
-	alpha = 100
+	invisibility = INVISIBILITY_ABSTRACT
```

Comment:
do you even need the hide proc? just set invisibility abstract in the object definition

Reply:
If we want to change how it is hidden in the future, or possibly enable some observers to toggle viewing these mapping helpers, or something, then we'd want to change it in one place.

hide() is called in more than one place.

Reply:
> If we want to change how it is hidden in the future, or possibly enable some observers to toggle viewing these mapping helpers, or something, then we'd want to change it in one place.

then they can readd this. theres no reason to make a proc for this when it can just be like this by default.